includes/Modules/Analytics/Advanced_Tracking/Measurement_Events/Measurement_Event.php

```diff
@@ -57,15 +57,20 @@ private function validate_config( $config ) {
 			'action'     => false,
 			'selector'   => false,
 			'on'         => false,
+			'metadata'   => false,
 		);
```

Comment:
It looks like we aren't using `category` anymore here (and I agree it makes sense to instead put `event_category` into `metadata`), so this should be removed.

Reply:
This issue has been resolved.